plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -1083,7 +1083,7 @@ public void testAlias()
                 "SELECT count(*) FROM orders");
     }
 
-    @Test
+    @Test(enabled = false) // https://github.com/trinodb/trino/issues/8380
```

Comment:
`// TODO ...`

Reply:
the inked issue is about `testAlias` (some other existing test) while here we're at `testSelectInformationSchemaForMultiIndexAlias`. Should the issue cover both?

Reply:
@findepi `testSelectInformationSchemaForMultiIndexAlias()` actually internally calls test `testSelectInformationSchemaColumns()` that resulted in the failure. Without `testSelectInformationSchemaForMultiIndexAlias()` also, test `testSelectInformationSchemaColumns()` can fail if `testAlias()` runs before. I have reported #8380 for tracking it and also raised PR #8381 to fix the same.